openslides_backend/services/datastore/extended_adapter.py

```diff
@@ -471,6 +471,6 @@ def _fetch_missing_fields_from_datastore(
             for fqid, fields in missing_fields_per_fqid.items()
         ]
         results = super().get_many(
-            get_many_requests, None, DeletedModelsBehaviour.ALL_MODELS, lock_result
+            get_many_requests, None, DeletedModelsBehaviour.NO_DELETED, lock_result
```

Comment:
@jsangmeister is this a bug in the current code? I think it is not necessary here to ask for all models

Reply:
Yes, this was probably a leftover from a previous state where I wanted to handle deleted models as well.